Makefile.am

```diff
@@ -38,6 +38,7 @@ SUBDIRS += examples
 endif
 
 EXTRA_DIST = \
+	.github/workflows/action.yml \
```

Comment:
Is that really something that belongs in the distribution source code tarball? Its Github specific.

Reply:
Probably not. Remove this?

Reply:
Yes please.

Reply:
@erikd, complete.